projects/packages/blaze/src/class-dashboard-config-data.php

```diff
@@ -81,6 +81,31 @@ public function get_data() {
 		);
 	}
 
+	/**
+	 * Gets the WordPress.com user's identity, if connected.
+	 *
+	 * @return array|bool
+	 */
+	protected function get_connected_user_identity() {
```

Comment:
Could we use `Jetpack_Tracks_Client::get_connected_user_tracks_identity()` here?

Reply:
Sure! I will make the change and test it. Thanks for the suggestion.

Reply:
I double-checked, and I still need to rename some of the attributes returned by the `get_connected_user_tracks_identity` method. I still need a method that acts as an intermediary and creates the dummy data (if a user is not connected), and add/rename the missing attributes.

What I can do here is use the `get_connected_user_tracks_identity` method instead of the `( new Manager() )->get_connected_user_data()`.

What do you think about that change?

Reply:
>  still need to rename some of the attributes returned by the get_connected_user_tracks_identity method.

Ah, in this case I suppose it's fine as it is today. 👍